src/client/pps/pps.proto

```diff
@@ -94,6 +94,13 @@ message Datum {
   bytes hash = 5;
 }
 
+message WorkerStatus {
+  string worker_id = 1 [(gogoproto.customname) = "WorkerID"];
+  string job_id = 2 [(gogoproto.customname) = "JobID"];
+  repeated pps.Datum data = 3;
+  google.protobuf.Timestamp started = 4;
```

Comment:
A comment about what `started` means?  When the worker launches or when the worker started processing this datum?